lib/inspec/base_cli.rb

```diff
@@ -240,7 +240,7 @@ def plain_text(msg)
 
     def suppress_log_output?(opts)
       return false if opts['reporter'].nil?
-      match = %w{json json-min json-rspec junit html yaml documentation progress} & opts['reporter'].keys
+      match = %w{json json-min json-rspec json-automate junit html yaml documentation progress} & opts['reporter'].keys
```

Comment:
should the 'automate' reporter also be listed here? or does that go through different logic

Reply:
Yeah, we ship the raw json up manually so it would be fine if something showed up on the cli before the post.